README.md

```diff
@@ -42,6 +42,37 @@ Use Funogram.Api module to communicate with Telegram API with the help of reques
 ```
 So, that is it! Use Intellisence-driven development approach to explore all Funogram features! For further learning you may take a look at sample Telegram bots built using this library: <a href="https://github.com/Dolfik1/Funogram/tree/master/Funogram.TestBot">Test Bot</a>, <a href="https://github.com/worldbeater/Memes.Bot/tree/master/Memes">Memes Bot</a>
 
+# Getting updates via webhooks
+If you want to use webhooks, you should start application with admin privileges.
```

Comment:
It is unclear why you need some "admin privileges". It could be a port requirement (*nix only, then?) or some silly HTTPS.sys idiosyncrasie (Windows-only, then?); not every OS/environment even has a concept of an "administrator".

Could you please clarify?

Reply:
It seems that admin privileges required only for windows system for HttpListener. I cannot claim this since I have not tested it on other systems.